app/src/main/java/de/westnordost/streetcomplete/settings/questselection/QuestSelectionFragment.kt

```diff
@@ -113,4 +128,16 @@ class QuestSelectionFragment
         questTypeOrderList.sort(questTypes)
         return questTypes.map { QuestVisibility(it, visibleQuestTypeController.isVisible(it)) }.toMutableList()
     }
+
+    private fun updateSubtitle() {
+        val subtitle = getSubtitle(requireContext(), questTypeRegistry, visibleQuestTypeController)
+        (requireActivity() as AppCompatActivity).supportActionBar?.subtitle = subtitle
```

Comment:
Ah, please don't do that. A fragment should not control its parent activity like that.

There is an interface called `HasTitle`. You could expand it with another field `val subtitle: String?`. The logic should happen in `FragmentContainerActivity`. It is probably necessary to add a public method to `FragmentContainerActivity` called something like `updateTitle` then. Or maybe better: An interface called `ContainsTitle` or similar that is implemented by `FragmentContainerActivity` that has this method. (Since any fragment could also be the child of anoter fragment)

The `QuestSelectionFragment` would then talk to its parent in the same pattern as all other Fragments in StreetComplete do:
```kotlin
private val parentTitleContainer: TitleContainer ? get() = 
    parentFragment as? TitleContainer ?: activity as? TitleContainer
```